src/tests/scenarios/test_iso_yielder_setting.cairo

```diff
@@ -827,9 +827,9 @@ mod FarmingClaimingReward {
     use super::{SLOT, VALUE, PROJECT_VALUE, PRICE, ONE_MONTH, TON_EQUIVALENT};
     use super::SpanPrintImpl;
 
-
-    #[test]
-    #[available_gas(19_000_000_000)]
+    // Costly test to activate locally only
+    //#[test]
+    //#[available_gas(19_000_000_000)]
```

Comment:
Why comment the test ?

Reply:
This test is very CI-time-consuming so it is meant to be a local test only.